### ISSUE TITLE
Solar Wind geometry not implemented error 

### ISSUE THREAD

**Issue by mtlam** [nanograv/PINT#1826]
Having this issue on my local machine and the notebook server. I'm taking a NANOGrav par file and trying to add in SWX parameters. Each is of the form:
```
SWXDM_0001   0.001   1
SWXP_0001   2
SWXR1_0001   <float>
SWXR2_0001   <float>
```
which should allow fitting some DM over that time range. But even with one of the above blocks, I get the following error:
`NotImplementedError: Solar Wind geometry not implemented for power-law index p <= 1`
I get this regardless of what value I put into the `SWXP` parameters.

I tried to diagnose with a test installation of PINT but running into other compatibility issues that prevent me from easily going forward. The source code itself looks correct from what I can tell but still having the problem.

**Comment by mtlam** [comment]
@dlakaplan did some digging and found that on loading a test model that `SWXP_0001` is shown in `m.free_params`. I tested and if I change the second line in the block above to `SWXP_0001   2   0` to explicitly include the frozen flag, then the code does work.

**Comment by mtlam** [comment]
Resolved in #1827.
